docs/modules/dev/pages/manual_build.adoc

```diff
@@ -148,16 +152,43 @@ apt-get install -y --no-install-recommends \
 .Compile
 [source,bash]
 ....
-cmake -Bbuild -DCMAKE_C_FLAGS=$CFLAGS -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_EXTENSIONS=OFF -G Ninja
+cmake -Bbuild -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_EXTENSIONS=OFF -G Ninja
 ninja -C build
 ....
 
-If compilation completes correctly, you can finally start Wolf
+If compilation completes correctly, you can finally start Wolf. The built binary can be found at `build/src/moonlight-server/wolf`
+Since Wolf is configured via a swoth of environment variables, it may be a good idea to lanch it via shell script.
+
+.runwolf.sh
+[source,bash]
+....
+#!/bin/bash
+
+: ${WOLF_CFG_FOLDER="config"}
+
+XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR=/tmp/sockets}" \
```

Comment:
The format for all those env variables is odd, I thought that this was possible:
```bash
# This will set XDG_RUNTIME_DIR if it wasn't set already
${XDG_RUNTIME_DIR:=/tmp/sockets}
```
or
```bash
# This will override XDG_RUNTIME_DIR if not set
XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR:-/tmp/sockets}"
```
You don't put `:` though and use the `=` which seems to work from a quick test on the CLI but I'm not sure if there's any pitfall when doing this..

Reply:
I'll do some tests tonight to see what's going on. looking at the patch, it does strike me as odd syntax and i think it probably should be `:=`.

Reply:
hey @robjh,

Just reading through this PR, how did the tests go?

Reply:
apologies @salty2011, this fell off my radar. Thank you for the reminder, i'll get to this within the next few days. If i dont, feel free to give me a poke on discord.

Reply:
okay! tests done :)
:- is the appropriate operator for those variables. I've updated the commit and rebased on the latest from the stable branch.